libs/langchain-openai/src/chat_models.ts

```diff
@@ -1190,6 +1207,8 @@ export class ChatOpenAI<
       seed: options?.seed,
       ...streamOptionsConfig,
       parallel_tool_calls: options?.parallel_tool_calls,
+      audio: this.audio,
```

Comment:
May be slightly safer to add these fields only if they are present - not sure how Azure or other model proxies will react to adding `audio: undefined`